src/screens/regimen_setup/currentProducts/style.ts

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components/native';
+import applyScale from '../../../utils/applyScale';
+
+export const Container = styled.View`
```

Comment:
give this component flex 1 for screen optimisation

Reply:
Changed it to flex:1 and removed width 100%